ecosystem/platform/server/app/views/layouts/_application.html.erb

```diff
@@ -29,6 +29,7 @@
     <link rel="icon" type="image/x-icon" href="/favicon.ico" sizes="any">
     <link rel="icon" type="image/svg+xml" href="/favicon.svg">
     <link rel="icon" type="image/image/png" href="/favicon.png">
+    <link rel="apple-touch-icon" type="image/image/png" href="/apple-touch-icon.png">
```

Comment:
I think the mimetype is wrong, shouldn't it just be `image/png` (ditto for the preceding line)?